configs/vision/dino_vit/offline/camelyon16.yaml

```diff
@@ -123,13 +123,13 @@ data:
             split: test
     dataloaders:
       train:
-        batch_size: &BATCH_SIZE ${oc.env:BATCH_SIZE, 16}
+        batch_size: &BATCH_SIZE ${oc.env:BATCH_SIZE, 32}
         shuffle: true
       val:
         batch_size: *BATCH_SIZE
       test:
         batch_size: *BATCH_SIZE
       predict:
         batch_size: &PREDICT_BATCH_SIZE ${oc.env:PREDICT_BATCH_SIZE, 64}
-        num_workers: 12 #multiprocessing.cpu_count
+        num_workers: ${oc.env:NUM_WORKERS, 12}
```

Comment:
i dont remember why do we have to  the `multiprocessing.cpu_count` is too much?

Reply:
removed `num_workers` here since it uses `multiprocessing.cpu_count()` by default